src/plugin.js

```diff
@@ -25,6 +25,8 @@ const splashPlay = function(options) {
     let player = this;
     let splashButton = document.createElement('button');
 
+    splashButton.setAttribute('aria-label', 'play button');
+
```

Comment:
@ojjasvijain remove extra line

Reply:
Without space , it was giving lint error . Its required 
Thanks